spec/lib/champaign_queue/clients/sqs_spec.rb

```diff
@@ -2,18 +2,40 @@
 
 describe ChampaignQueue::Clients::Sqs do
   context "with SQS_QUEUE_URL" do
-    it "delivers payload to AWS SQS Queue" do
+    let(:resp_body) do
+      %{<?xml version="1.0"?>
+          <SendMessageResponse xmlns="http://queue.amazonaws.com/doc/2012-11-05/">
+            <SendMessageResult>
+              <MessageId>918aba5a-b70f-4e31-9905-ba02000fcdaa</MessageId>
+              <MD5OfMessageBody>9bb58f26192e4ba00f01e2e7b136bbd8</MD5OfMessageBody>
+            </SendMessageResult>
+            <ResponseMetadata>
+              <RequestId>cd1c9703-d46f-555b-a927-d94c5e569d55</RequestId>
+            </ResponseMetadata>
+        </SendMessageResponse>}
+    end
```

Comment:
Isn't this basically the use case for VCR so that we don't need to have stuff like this coded into the spec?